src/utils/public-ip.ts

```diff
@@ -56,20 +58,32 @@ async function pokeStun(): Promise<string> {
 }
 
 export async function getPublicNetworkInfo(): Promise<any> {
+  const { flags } = getContext()
+  const isSymonMode = isSymonModeFrom(flags)
+
   try {
-    const ip = await pokeStun()
+    let ip = await pokeStun()
     const response = await sendHttpRequest({
       url: `http://ip-api.com/json/${ip}`,
     })
     const { country, city, isp } = response.data
-    publicNetworkInfo = { country, city, isp }
-    log.info(
-      `Monika is running from: ${publicNetworkInfo.city} - ${
-        publicNetworkInfo.isp
-      } (${ip}) - ${hostname()} (${getIp()})`
-    )
+
+    // do we reveal ip and location details?
+    if (flags.verbose || isSymonMode) {
+      publicNetworkInfo = { country, city, isp }
+      publicIpAddress = ip // store public ip
+      log.info(
+        `Monika is running from: ${publicNetworkInfo.city} - ${
+          publicNetworkInfo.isp
+        } (${ip}) - ${hostname()} (${getIp()})`
+      )
+    } else {
+      log.info('network connectivity ok')
+      publicNetworkInfo = { country: '', city: 'Earth', isp: 'localhost' } // Hide specifics
+      ip = '0.0.0.0'
```

Comment:
i think it's better to use something like `x.x.x.x` to tell that it's not an actual IP Address